appsensor-ui/src/main/java/org/owasp/appsensor/AppsensorUiApplication.java

```diff
@@ -59,7 +59,7 @@ public EmbeddedServletContainerCustomizer containerCustomizer(){
         public void customize(ConfigurableEmbeddedServletContainer container) {
         	ErrorPage error400Page = new ErrorPage(HttpStatus.BAD_REQUEST, "/400.html");
             ErrorPage error401Page = new ErrorPage(HttpStatus.UNAUTHORIZED, "/401.html");
-	    ErrorPage error403Page = new ErrorPage(Httpstatus.FORBIDDEN, "/4.3.html");
+	    ErrorPage error403Page = new ErrorPage(Httpstatus.FORBIDDEN, "/403.html");
```

Comment:
This doesn't compile, it should be HttpStatus.FORBIDDEN but not Httpstatus.FORBIDDEN